src/key.rs

```diff
@@ -608,6 +618,247 @@ impl HasByteSource for SecretAsymmetricKey {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub enum SigningKey {
+    SodiumOxideEd25519(SodiumOxideEd25519SecretAsymmetricKey),
+    RingEd25519(RingEd25519SecretAsymmetricKey),
+}
+
+impl StorableType for SigningKey {}
+
+impl From<SigningKey> for Key {
```

Comment:
Don't understand this From implementation.

1. What code requires a signing_key.into() to turn it into a Key?
2. Why is the implementation converting the SigningKey into a public key before wrapping it as a key?

Reply:
Can't remember what exactly it was originally for but it's no longer needed, removed